optimum/utils/input_generators.py

```diff
@@ -50,8 +50,8 @@ def wrapper(*args, **kwargs):
     "sequence_length": 16,
     "num_choices": 4,
     # image
-    "width": 224,
-    "height": 224,
+    "width": 64,
```

Comment:
224 was for imagenet, not sure how the export for vision models will work there

Reply:
I modified the default dummy inputs width and height as the validation step comparing the reference and the ONNX model is quite memory intensive for stable diffusion model (leading to OOM error). I fixed the `LayoutLMv3OnnxConfig` to get the `image_size` from the config as it should be fixed. For the other architectures, I'm expecting that their export should work for any weight / height default values, if not we should fix their corresponding `OnnxConfig` no ?

Reply:
Yes discussing with Michael this should be fine. I would recommend using tiny models for the tests.

Reply:
Yes it's already the case for the tests ! It's more for common users using the CLI with classic stable diffusion models.